### DISCUSSION TITLE
Available for MS Windows?

### DISCUSSION THREAD

**Discussion by MikeBee** [kmwallio/ThiefMD#152] [General]
Can ThiefMD be compiled for Windows 10?  It looks great, so would love to use it.

**Comment by kmwallio** [comment]
If you run [WSLg](https://github.com/microsoft/wslg), you can potentially use the PPA (Ubuntu 21.04 or 20.10 may work).

```
sudo add-apt-repository ppa:thiefmd/thiefmd
sudo apt-get update
sudo apt-get install com.github.kmwallio.thiefmd
```

At the moment, [webkit2gtk](https://github.com/msys2/MINGW-packages/issues/3484#issuecomment-715288124) is not available on Windows, so direct compilation isn't available.

It might be possible on a future windows build to remove the built-in browser and instead have a web server hosting the preview. Then just use the user's web browser.
